src/pages/404.tsx

```diff
@@ -0,0 +1,156 @@
+import { Box, Flex, Image, Text, useMediaQuery } from '@chakra-ui/react';
+
+import Footer from '@/uikit/Footer';
+import Header from '@/uikit/Header';
+
+function DescriptionMobile() {
+  return (
+    <Text as='h3' fontWeight='light'>
+      Whoops, you’ve wandered far, even Pluto is in your rearview! Seems like
+      the page you’re looking for has drifted off-course
+    </Text>
+  );
+}
+
+function DescriptionDesktop() {
+  return (
+    <>
+      <Text as='h3' fontWeight='light'>
+        Whoops, you’ve wandered far, even Pluto is in your rearview!
+      </Text>
+      <Text>Seems like the page you’re looking for has drifted off-course</Text>
+    </>
+  );
+}
+
+export default function Page404() {
+  const isMobile = useMediaQuery('(max-width: 768px)')[0];
```

Comment:
const [isMobile] = useMediaQuery('(max-width: 768px)');

Reply:
There is no difference between isMobile and [isMobile]; it is just an array and how we access it right? What is your concern?

Reply:
`useMediaQuery('(max-width: 768px)')[0];`

Reply:
updated @asaadam